packages/web3/src/index.ts

```diff
@@ -29,5 +29,5 @@ export * from './token'
 
 export * from './constants'
 export * as web3 from './global'
-
+export * as codec from './codec'
```

Comment:
As discussed, we don't want to expose new functions via root `web3/src/index.ts`.

Reply:
Got it :+1: